### ISSUE TITLE
Accept quoted column names in PRIMARY KEY declaration

### ISSUE THREAD

**Issue by ravenAtSafe** [olavloite/spanner-jdbc#7]
The following create statement throws a `java.sql.SQLException: Error while parsing sql statement` error because quoted identifiers are not supported within the PRIMARY KEY.

```
create table `SomeTableName` (
`creation_instance` int64
) PRIMARY KEY (`creation_instance`)

```

The following **does** work
```
create table `SomeTableName` (
`creation_instance` int64
) PRIMARY KEY (creation_instance)
```

**Comment by olavloite** [comment]
Caused by JSQLParser/JSqlParser#491 which has now been fixed

**Comment by olavloite** [comment]
Fixed in version 0.12.
